LayoutTests/imported/w3c/web-platform-tests/mixed-content/gen/top.http-rp/opt-in/img-tag.https.html

```diff
@@ -11,6 +11,10 @@
   </head>
   <body>
     <script>
+      if (window.internals) {
+          internals.registerDefaultPortForProtocol(8800, "http");
+          internals.registerDefaultPortForProtocol(9443, "https");
+      }
```

Comment:
We cannot really do that here without making a WPT PR upstream, and it would not be accepted since this is WTR only.
We should probably move it to testharnessreport.js , probably only if the location port is 8800 or 9443.

Reply:
Whoops, I missed removing this one, thanks. See below for more details about this. I'll move this logic into `testharnessreport.js` so we have the correct behavior in other tests that need it.